src/modals/share.js

```diff
@@ -31,7 +32,12 @@ export default class Share {
     this.modal.close()
   }
 
-  openPopup () {
+  openPopup (targetName) {
+    if(targetName === 'facebook') {
+      Telemetry.add(Telemetry.SHARE_FACEBOOK)
```

Comment:
These events are triggered on each share modal opening and closing, and on app initialization :upside_down_face:

Reply:
i dont reproduce on app init @amatissart